src/services/usersService.js

```diff
@@ -6,13 +6,61 @@ import { history } from "@helpers/history";
 import validateEmail from "@validations/user/validateEmail";
 import validateFieldLength from "@validations/validateFieldLength";
 import validateIsEmptyObject from "@validations/validateIsEmtpyObject";
+import validateLogin from "@validations/user/validateLogin";
+import validatePassword from "@validations/user/validatePassword";
 import validatePhone from "@validations/user/validatePhone";
 
 class UsersService {
   constructor() {
     this.currentUser = store.getState().currentUser.value;
   }
 
+  async login(data) {
+    const { login, password } = data;
+
+    if (!login?.length || !password?.length) {
+      return "The login and password fields must not be empty.";
```

Comment:
since the method is async (means, it returns a promise), it's better to throw an exception here, so it will be translated to promise reject

Reply:
Done. But I also thought, why don't we just throw a bug like that:
`throw new Error("Failed to create user");`